governance/paper-process.md

```diff
@@ -0,0 +1,235 @@
+
+### Tasking assignment
+
+Contributors should highlight and assign themselves to various areas of the
+outline.  This allows contributors to focus on a few areas and identify others
+where more contributors is needed.  If using a google doc for the paper,
+assignments may be done through the "assign to me" feature after highlight the
+area or by using the "+foo@bar.com" comment feature for the assignee's email.
+Once task assignment is complete, contributors can begin content rough-in.
+
+### Content rough-in
+
+Assignees can provide content rough-in in a variety of ways.  They may provide
+raw content in the form of phrases, disjointed paragraphs, bullets on the topic,
+or draft content in the form of completed paragraphs.  If leveraging draft
+content, placing this content in quotes helps other reviewers know it is
+'near-complete' and it should be reviewed as a holistic section.  
+
+#### Opinions
+
+There may be situations where multiple differing views on the subject matter or
+controls implementation are expressed.  Authors should make every effort to
+discover the industry standard or practice and document it. If none exists
+and the suggested content cannot be collaboratively resolved, it is the duty
+of the authors to capture the different views with references, benefits, and
+drawbacks or contingencies in order to provide the reader with enough
+information to reach their own conclusion.
+
+#### References
+
+It is important that any references used to generate paper content be cited
+appropriately through footnotes and within the references section.  This
+permits the reader to not only dig deep into a specific reference area but also
+provides source content and is a matter of good practice.  
+
+### Collaborative
+
+Contributors now review all content roughed-in and transition raw content into
+draft content.  They may begin commenting on sections, expanding ideas, refining
+arguments, and providing citations.  Contributors are encouraged to discuss
+content specific items in the doc through comments whereas larger themes and
+writing decisions should be discussed in the corresponding slack channel.  This
+keeps the doc clean as well as documents thematic decisions in the primary
+collaboration & coordination channel.
+
+### Executive summary and content wrap-up
+
+If the paper exceeds five (5) pages, an Executive summary commensurate with the
+length of the document (but not to exceed 3 pages) should be written.  Content,
+comments, and other discussions should be closed out and finalized at this time.
+
+### Narrative voice
+
+No more than three contributing individuals should be assigned for narrative
+voice. The narrative voice is a semi-final pass of the paper to ensure it reads
+as a single, unified voice. It should ensure:
+
+* the language origin is consistent throughout the document (lang_en or
+  lang_us),
+* phrasing is similar (caddy corner not mixed in with kitty corner),
```

Comment:
The reference of `caddy corner` vs `kitty corner` is confusing. I think you mean consistent flow of the language.

Reply:
https://brians.wsu.edu/2016/05/31/caddy-corner/

Reply:
Your reference only makes it more confusing: it explicitly discourages `caddy corner`. 

Back to my general comment, the goal of writing to a diverse community is to make the messaging clear. If readers have to use a dictionary, the writing is not done right.

Reply:
Shall we add something stating that regional slang will be, where possible, not in documents?  I am very careful when speaking across geos to avoid cultural references (not perfect, but I do my best).

I think for the above phrasing, can we just say "the language is consistent in tone throughout the document."  Honestly, I can take many different authors and do a once-over (or thrice over) and get a consistent tone, so I'm not entirely sure this is necessary with an appropriate editor.

Reply:
As important as an English speaker would be a non-English native speaker as a reviewer to ask on anything confusing.  English is weird and confusing.  Although this kinda argues for more people the merrier, from a clarity/understanding perspective.